py3-minidb.yaml

```diff
@@ -0,0 +1,78 @@
+# Generated from https://pypi.org/project/minidb/
+package:
+  name: py3-minidb
+  version: 2.0.8
+  epoch: 1
+  description: "Simple SQLite3 store for Python objects"
+  copyright:
+    - license: ISC
+  dependencies:
+    provider-priority: 0
+
+vars:
+  pypi-package: minidb
+  import: minidb
+
+data:
+  - name: py-versions
+    items:
+      3.10: '310'
+      3.11: '311'
+      3.12: '312'
+      3.13: '300'
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - py3-supported-python
+      - py3-supported-setuptools
+      - py3-supported-gpep517
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: b18571efe558491f692d9a8413a91772c920435e
+      repository: https://github.com/thp/minidb
+      tag: ${{package.version}}
+
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    description: ${{vars.pypi-package}} installed for python${{range.key}}
+    dependencies:
+      runtime:
+        - py${{range.key}}-sqlite3
+        - py${{range.key}}-setuptools
+        - py${{range.key}}-gpep517
+      provides:
+        - py3-${{vars.pypi-package}}
+      provider-priority: ${{range.value}}
+    pipeline:
+      - uses: py/pip-build-install
+        with:
+          python: python${{range.key}}
+      - uses: strip
+    test:
+      pipeline:
+        - uses: python/import
+          with:
+            python: python${{range.key}}
+            imports: |
+              import ${{vars.import}}
+
+  - name: py3-supported-${{vars.pypi-package}}
+    description: meta package providing ${{vars.pypi-package}} for supported python versions.
+    dependencies:
+      runtime:
+        - py3.10-${{vars.pypi-package}}
+        - py3.11-${{vars.pypi-package}}
+        - py3.12-${{vars.pypi-package}}
+        - py3.13-${{vars.pypi-package}}
+
+update:
+  enabled: true
+  manual: false
+  github:
+    identifier: thp/minidb
```

Comment:
the project doesn't do active release so we need to use `use-tag: true` here in update block.